### PR TITLE
utilities and list

### PR DESCRIPTION
**Pull request by Ramandeep9898**

### CODE REVIEW

File: style/comman.css [hunk 1]
```diff
@@ -26,6 +27,9 @@
   justify-content: flex-end;
 
 }
+.code{
+    /* border: 1px solid var(--gray-text); */
+}
```
Comment (ashishkatiyar2404) [inline style/comman.css:32]: remove these comments

File: style/components/list.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    gap: 32px;
+}
+.list-divider{
+    height: 0px;
+    border-bottom: 1px solid;
+    color: rgba(0, 0, 0,.12);
+}
+.input-ckeckbox{
+    width: 18px;
+    height: 18px;
```
Comment (ashishkatiyar2404) [inline style/components/list.css:53]: use other units(rem) when the value is more than 10px.

File: style/components/utilities.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    color: #007bff!important;
+}
+.green-color{
+    color: #28a745!important;
+}
+.danger-color{
+    color: #dc3545!important;
+}
+.info-color{
+    color: #17a2b8!important;
```
Comment (ashishkatiyar2404) [inline style/components/utilities.css:50]: you use root variables for colors but not here?

**Comment by ashishkatiyar2404** [comment]
your code is good. Easy to read.
also, you use good names for classes which is pretty much understandable to me.
You can merge💯,  I just commented on a few things.

File: style/components/utilities.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+.gray-color{
+    color: var(--gray-text);
+}
+.blue-color{
+    color: #007bff!important;
+}
+.green-color{
+    color: #28a745!important;
+}
+.danger-color{
```
Comment (anshulraheja) [inline style/components/utilities.css:46]: use root variables for colors

**Review by anshulraheja — APPROVED** [review]
clean code. you can merge

**Review by m26unkwn — APPROVED** [review]
Good to merge

File: style/components/list.css (hunk 2 above)
Comment (Ramandeep9898) [reply style/components/list.css:53]: done!!

File: style/components/utilities.css (hunk 3 above)
Comment (Ramandeep9898) [reply style/components/utilities.css:50]: okay

File: style/components/utilities.css (hunk 4 above)
Comment (Ramandeep9898) [reply style/components/utilities.css:46]: yeas thanku

**Pull request merged**